### ISSUE TITLE
cannot add alternateIdentifier to dataset

### ISSUE THREAD

**Issue by jeanetteclark** [NCEAS/metacatui#1220]
user wrote in on this ticket describing the issue https://support.nceas.ucsb.edu/rt/Ticket/Display.html?id=19462

I confirmed the same behavior - I tried to add an alternateIdentifier (a DOI) to his dataset and got a "fix the flagged errors below" error message but nothing was flagged.

**Comment by amoeba** [comment]
I'll take a look at this today.

**Comment by laurenwalker** [comment]
Thanks Bryce. You can use Zenhub to move the tickets tto the In Progress column rather than commenting on the issue itself.

Also pls close tickets when they're done. I know they'll get autoclosed when merged to master but closing them makes it easy to track what needs to be done. Thanks

**Comment by amoeba** [comment]
Can't reproduce on tip of dev-2.11 branch.

Did this in the editor:

<img width="281" alt="Screen Shot 2020-04-17 at 11 21 41 AM" src="https://user-images.githubusercontent.com/563/79606196-b00bb700-809d-11ea-8726-5909c93dbeda.png">

And saw this when I saved it:

<img width="354" alt="Screen Shot 2020-04-17 at 11 21 52 AM" src="https://user-images.githubusercontent.com/563/79606216-b863f200-809d-11ea-99a6-eac5341023a9.png">

I grepped around the commit messages and see some work has gone in here and there prior to when this bug was filed around handling validation but couldn't find anything that stuck out a a reason this might have gone away since December of 2019. The relevant code WRT `alternateIdentifier`, as far as I can tell, hasn't been touched in 7 months to 2 years.

I also confirmed this all worked on 2.10.0 by testing on test.arcticdata.io and it also works there.

**Comment by amoeba** [comment]
Thanks @laurenwalker, will do on both counts.

**Comment by laurenwalker** [comment]
Hmm, December 2019 is still pretty recent, and if Jeanette was able to confirm the bug, I'm not so certain we should close the ticket yet. Sounds like there may be more debugging to do.

I can take a whack at trying to reproduce, too. If neither of us can reproduce, we can always keep open with a 'cannot reproduce' label. That's typically what I've done in these situations

**Comment by amoeba** [comment]
Sounds good.

**Comment by jeanetteclark** [comment]
I can no longer reproduce

**Comment by laurenwalker** [comment]
I was able to reproduce this bug and fixed it.

I reproduced by trying to edit a copy of the EML that originally caused this bug (from the linked support ticket). The issue was not actually about alternate identifiers, but EMLParties.

Probably about 30% of the time, reported bugs are not actually about the issue described in the ticket/email. Haha. So it helps to work with the same conditions that the user had when the bug happened.

Thanks Jeanette and Bryce for looking into it.
